azure-pipelines.yml

```diff
@@ -670,7 +670,7 @@ jobs:
       set -e
       . util/build_consts.sh
       module load "xilinx/vivado/$(VIVADO_VERSION)"
-      ci/scripts/run-fpga-tests.sh cw310 fpga_cw310_rom_ext_tests || { res=$?; echo "To reproduce failures locally, follow the instructions at https://opentitan.org/book/doc/getting_started/setup_fpga.html#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
+      ci/scripts/run-fpga-tests.sh hyper310 fpga_hyper310_rom_ext_tests || { res=$?; echo "To reproduce failures locally, follow the instructions at https://opentitan.org/book/doc/getting_started/setup_fpga.html#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
```

Comment:
Oops, it's missing the artifacts from (and dependency on) the chip_earlgrey_cw310_hyperdebug job. :)

Reply:
I am not a 100% sure I believe we have tests that are defined for fpga_cw310_rom_ext and not fpga_hyper310_rom_ext_tests (like the cryptotests). With this changes, those tests will not run anymore, but maybe they don't need to run on earlgrey_es?

Reply:
There are some tests defined that use `fpga_cw310_rom_ext`.  They're located in `//sw/device/silicon_owner/bare_metal` and `//sw/device/tests/crypto`.  However, in the `run-fpga-tests` script, it appears that an `fpga_tags` argument (arg 2) of `fpga_{whatevs}_rom_ext_tests` sets the pattern to `//sw/device/silicon_creator/rom_ext/e2e/...`, so this pipeline stage was never running those other tests in the first place.

Reply:
AFAICT, the following tests are not currently being run in CI:
```
//sw/device/silicon_owner/bare_metal:rom_ext_virtual_bare_metal_virtual_boot_test
//sw/device/silicon_owner/bare_metal:rom_ext_virtual_ottf_bl0_virtual
//sw/device/tests/crypto:otcrypto_hash_test
//sw/device/tests/crypto:otcrypto_export_test
```

The change in this PR does not affect whether or not these tests run.

It would be nice to run them on `earlgrey_es_sival`.  We should make sure they run on `master`:
- The bare metal test is a post-ROM_EXT test that checks if we can run owner code that is mostly separate from our existing test framework.  We could argue about whether the this test truly provides value now that there are ROM_EXT E2E tests in place.
- The crypto tests check that the crypto library distributable build artifact is functional.

Reply:
If you want to run these jobs on the earlgrey_es_sival branch, I think a new FPGA job will be needed. I will double-check that they run on master (or at least will run once the CI orchestrator is merged)